components/UserDashboard/ProductionButton.jsx

```diff
@@ -20,11 +20,11 @@ export default function ProductionButton({ navigation, production }) {
             <View className="w-max flex flex-row mt-2">
                 {Object.keys(production.participants)
                     .slice(0, 5)
-                    .map((id) => {
+                    .map((id, index) => {
                         return (
                             <>
                                 <ProfilePicture
-                                    key={id}
+                                    key={index}
```

Comment:
Don't think this has fixed it :/

Reply:
seems like it has for me. when does the error come up? is it only for lots of participants?

Reply:
actually surely the error is cos the view didn't have a key so I've given them keys now